### PR TITLE
Don't install select2 via composer, fix composer.json

### LINKED ISSUE #745: select2 installed to js folder instead of vendor
Is there any reason to install select2 as a composer package?

Why? For example, let's assume that we start with a composer-based Concrete project [like this](https://github.com/mlocati/concretecms-skeleton).

If we add `concrete5-community-store/community_store` as a composer dependency, the package will be installed in
```
/web/packages/community_store
```
(that is, under the `web` webroot), but its dependencies will be installed in
```
/vendor
```
(that is, outside the webroot).

But Community Stores assumes that the composer dependencies are installed under
```
/web/packages/community_store/vendor
```

The solution is simply to remove select2 from the composer dependencies, and install it exactly the same way we already install other npm packages.

### PR DESCRIPTION
**Pull request by mlocati**
Close #745

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  }
-
-}
+    "name": "concrete5-community-store/community_store",
+    "description": "An open, free and community developed eCommerce system for Concrete CMS",
+    "type": "concrete5-package",
+    "homepage": "https://github.com/concretecms-community-store/community_store",
+    "license": "MIT",
+    "require": {
+        "php": ">=5.6"
```
Comment (mlocati) [inline composer.json:8]: Is PHP 5.6 the minimum PHP version?

Comment (Mesuva) [reply composer.json:8]: I think that would still be correct, but it probably makes sense to have it 7.4, to better align it with V8 requirements

Comment (mlocati) [reply composer.json:8]: Isn't Community Store [compatible with concrete5 v8.5](https://github.com/concretecms-community-store/community_store/blob/v2.4.8.5/controller.php#L22)?
If so, the minimum PHP version is 5.5.
If Community Store is only compatible with ConcreteCMS 9+, the minimum PHP version is [7.3](https://github.com/concretecms/concretecms/blob/9.0.0/concrete/composer.json#L28).

Comment (Mesuva) [reply composer.json:8]: It should be compatible with version 8 in general.
I think I set it to 5.6, as there was some code in there that wasn't playing nicely with 5.5.

**Pull request merged**